apps/annotations/views/tag_views.py

```diff
@@ -46,6 +47,19 @@ def get_success_url(self):
     def form_valid(self, form):
         form.instance.team = self.request.team
         form.instance.created_by = self.request.user
+        form.instance.name = unicodedata.normalize("NFC", form.instance.name)
```

Comment:
This is a nice touch - what made you add it?

Reply:
it caused weird bugs elsewhere in the platform (ex the filter widget) so I tracked it down-- I didn't know the emojis would need special handling but good to know!